Support for the TBA 6.5 (#954)
teloxide/teloxide: crates/teloxide-core/CHANGELOG.md

  - `video_chat_ended`
  - `web_app_data` 
- `is_delete_chat_photo`, `is_group_chat_created`, `is_super_group_chat_created`, `is_channel_chat_created` functions to `Message` ([#982][pr982])

> - Add `can_send_audios`, `can_send_documents`, `can_send_photos`, `can_send_videos`, `can_send_video_notes`, and `can_send_voice_notes` for `ChatPermissions`, `Restricted` and `ChatPermissionsRaw` ([#954][pr954])
> - Add the `use_independent_chat_permissions` field to `RestrictChatMember` and `SetChatPermissions` ([#954][pr954])
> - Add `user_chat_id` field to `ChatJoinRequest` ([#954][pr954])
> - Add `ChatShared` kind to `MessageKind` ([#954][pr954])
> - Add `shared_chat` instance function to `Message` to get the `ChatShared` ([#954][pr954])
> - Add `KeyboardButtonRequestChat` and `ChatShared` to types ([#954][pr954])
> - Add `RequestChat` to `ButtonRequest` ([#954][pr954])
> - Add `UserShared` kind to `MessageKind` ([#954][pr954])
> - Add `shared_user` instance function to `Message` to get the `UserShared` ([#954][pr954])
> - Add `KeyboardButtonRequestUser` and `UserShared` to types ([#954][pr954])
> - Add `RequestChat` to `ButtonRequest` ([#954][pr954])

Reviewer: ```suggestion
- Support for TBA 6.5  ([#954][pr954])
  - Add `can_send_audios`, `can_send_documents`, `can_send_photos`, `can_send_videos`, `can_send_video_notes`, and `can_send_voice_notes` to `ChatPermissions` and `Restricted`
  - Add `use_independent_chat_permissions` optional parameter to `restrict_chat_member` and `set_chat_permissions`
  - Add `user_chat_id` field to `ChatJoinRequest`
  - Add `KeyboardButtonRequestChat` and `ChatShared` types
  - Add `RequestChat` variant to `ButtonRequest`
  - Add `ChatShared` variant to `MessageKind`
  - Add `shared_chat` method to `Message`
  - Add `KeyboardButtonRequestUser` and `UserShared` types
  - Add `RequestUser` variant to `ButtonRequest`
  - Add  `UserShared` variant to `MessageKind`
  - Add `shared_user` method to `Message`
```
Some stylistic choices to use more consistent and commonly used language

[pr850]: https://github.com/teloxide/teloxide/pull/850

### Removed

> - Remove `can_send_media_messages` from `ChatPermissions` and `ChatPermissionsRaw` ([#954][pr954])

Reviewer: ```suggestion
- Remove `can_send_media_messages` from `ChatPermissions` ([#954][pr954])
```

`ChatPermissionsRaw` is an implementation details.

> - Remove `ChatMemberKind::can_send_media_messages` instance function ([#954][pr954])

Reviewer: ```suggestion
- Remove `ChatMemberKind::can_send_media_messages` instance function ([#954][pr954])
```
This was reverted
